vuestorefront/README.rst

```diff
@@ -22,6 +22,26 @@ forked module:
 * implemented ``products`` ``category_slug`` filter.
 * moved Product schema domain method to ``product.template`` to be extendable.
 
+Addresses
+---------
+
+* ``address_type`` filter renamed to ``address_types``, because its a list of
+  types, not a single type.
```

Comment:
Now that I already changed this one, can I keep it? Shouldn't be too hard to update as that naming is very confusing (and I miss understood this filter/arg at first, because of it..). Will try not to do such changes to keep it backwards compatible:)

Reply:
I fully understand, but I would suggest moving this one back as well. Let's not add another issue if we don't have to. Thank you!

Reply:
OK, but if we can't do such little changes, we will have bigger problems down the road..:)